### PR TITLE
Feat : Docker-compose launch method

### LINKED ISSUE #3: Docker-compose method for local development

### LINKED ISSUE #4: Docker image

### PR DESCRIPTION
**Pull request by Tioborto**
##### SUMMARY

* Docker image (#4)
* Docker-compose launch method (#3)

##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +103,7 @@ Please type hint your code, or add types to the code you change.
 
 ## License
 
-Pydantic-Vault is available under the [MIT license](./LICENSE).
+Raddar is available under the [MIT license](./LICENSE).
```
Comment (nymous) [inline README.md:106]: xD

File: .env_template [hunk 2]
```diff
@@ -0,0 +1 @@
+API_KEY=""
```
Comment (nymous) [inline .env_template:2]: Maybe add a comment to explain that this is the GitHub webhook secret? (or rename the variable)

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+WORKDIR /raddar/
+
+# Install Poetry
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | POETRY_HOME=/opt/poetry python && \
+    cd /usr/local/bin && \
+    ln -s /opt/poetry/bin/poetry && \
+    poetry config virtualenvs.create false
+
+# Copy poetry.lock* in case it doesn't exist in the repo
+COPY ./pyproject.toml ./poetry.lock* /raddar/
```
Comment (nymous) [inline Dockerfile:14]: I wonder if we should specify the full path `/raddar/` every time or instead rely on the `WORKDIR` chosen above

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+RUN poetry install --no-root --no-dev && \
+    mkdir /raddar/results
+
+COPY ./raddar /raddar/raddar/
+ENV PYTHONPATH=/raddar
```
Comment (nymous) [inline Dockerfile:20]: Woah, this works?! Where did you find this trick?

**Review by nymous — APPROVED** [review]
Nice, I only have comments :100:

File: Dockerfile (hunk 3 above)
Comment (Tioborto) [reply Dockerfile:14]: I prefer to specify the full path when changing the workdir

File: Dockerfile (hunk 4 above)
Comment (Tioborto) [reply Dockerfile:20]: I don't know. It's the base of FastAPI dockerfile

**Pull request merged**